config/config.html

```diff
@@ -19,20 +20,20 @@
           <td>[[ preview(value) ]]</td>
           <td class="remove">
             <span class="clickable" ng-click="del(key)">
-              <i class="icon-remove" alt="remove"></i>
+              <i class="fa fa-times" style="color:red" alt="remove"></i>
```

Comment:
Can you use an `icon-red` class, or `text-danger`, which will be the BS3 equivalent.